mmv1/products/vertexai/Index.yaml

```diff
@@ -131,6 +131,16 @@ properties:
               performed. Exact reordering is a procedure where results returned by an
               approximate search algorithm are reordered via a more expensive distance computation.
               Required if tree-AH algorithm is used.
+          - !ruby/object:Api::Type::String
+            name: 'shardSize'
+            description: |-
+              Index data is split into equal parts to be processed. These are called "shards".
+              The shard size must be specified when creating an index. The value must be one of the followings:
+              * SHARD_SIZE_SMALL: Small (2GB)
+              * SHARD_SIZE_MEDIUM: Medium (20GB)
+              * SHARD_SIZE_LARGE: Large (50GB)
+            immutable: true
+            default_value: 'SHARD_SIZE_MEDIUM'
```

Comment:
A bit hesitant about adding an immutable field with a default value. If existing resources have defaulted to a different value, a diff is created that could potentially cause a destroy+replace if not corrected. 

How confident are we that this is the default value for all cases? I could not find documentation stating so. 
A safer approach would be to label `default_from_api: true` instead where Terraform would defer to the API's value instead of proposing its own. Thoughts?

Reply:
Oh I didn't know about `default_from_api`. That sounds great to me! I'll update the yaml file. Thank you

As for the documentation, I couldn't find any doc about the default value either. So I created https://issuetracker.google.com/issues/290565133.

Reply:
Thanks!

Reply:
Updated it in d2f1319...b36bed6